scripts/zones/Tahrongi_Canyon/Zone.lua

```diff
@@ -132,4 +133,19 @@ function onEventFinish( player, csid, option)
     if (csid == 35) then
         lightCutsceneFinish(player); -- Quest: I Can Hear A Rainbow
     end
+end;
+
+function isHabrokWeather(weather)
+    if (weather == WEATHER_DUST_STORM or weather == WEATHER_SAND_STORM or weather == WEATHER_WIND or weather == WEATHER_GALES) then
+        return true;
+    end
+    return false;
```

Comment:
can reduce this all to `return weather == WEATHER_DUST_STORM or weather == WEATHER_SAND_STORM or ...etc`

Reply:
Done